pink/configuration.py

```diff
@@ -89,12 +105,28 @@ def __init__(
         self.model = model
         self.q = q_readonly
         self.tangent = model.tangent
-        #
+
+        self.collision_model = collision_model
+        self.update_collision = self.collision_model is not None
+
+        if self.collision_model is not None:
```

Comment:
This looks like work users should do when they prepare their collision models, rather than relying on the Configuration constructor to do it internally.

* We could for instance move it to a utility function in `utils.py` with a prototype like `process_collision_pairs(collision_model, srdf_path)`.
* And/or (independent from Pink) we could process collision pairs automatically in robot_descriptions' [`load_in_pinocchio`](https://github.com/robot-descriptions/robot_descriptions.py/blob/943ad0512ad63043f5f774fe141715cc4873e20b/robot_descriptions/loaders/pinocchio.py#L33) function, doing something like "if the description has an `SRDF_PATH`, then process its collision pairs".

Reply:
I anticipate the second approach, since (at least for me) a `.srdf` file is subject to a frequent dynamic change, since a set of excluded pairs strongly depends on the performed task, and might be tweaked to find suitable performance/accuracy trade-off.

With that being said, I find the first approach suitable, and would implement it, hope I got your idea correctly